suse_migration_services/units/reboot.py

```diff
@@ -25,6 +27,22 @@ def main():
     DistMigration reboot with new kernel
     """
     try:
+        # Note:
+        # After the migration process is finished, the system reboots.
+        # The system will boot either in a migrated system or in a
+        # roll-backed system one in case the migration failed.
+        # In order to avoid rebooting, the migration must have failed
+        # and /etc/system_migration_debug file must exist.
+        debug_file = '/etc/system_migration_debug'
```

Comment:
In addition you are searching the debug file on the migration system. I thought we want to lookup that file on the system we want to migrate such that we have a better control to enable/disable debugging. So I would have expected that file to be searched in:

```
migration_debug_file = os.sep.join(
    [root_path, '/etc/system_migration.yaml']
)
```

and maybe support some conf style:

```yaml
debug: true
```

Reply:
It would give us more flexibility to control other behavior of the migration when needed. I'm sure there will be some in the future. Reading yaml in python is easy so I suggest to do it that way

Reply:
I understood from the reboot issue conversation that the path would be '/etc/system_migration_debug'

Reply:
It has to be [root_path, '/etc/system_migration_debug']. The user cannot put any files into the migration system. If we walk through the progression we have.

- User initiates migration
- Migration fails and user ends back in system pre-migration
- If the user want to debug the migration they only can access /etc in the system to be migrated

Reply:
yes that's what I think too